www-apps/grafana-bin/files/grafana.initd.3

```diff
@@ -9,19 +9,21 @@ depend() {
 start() {
 	ebegin "Starting grafana server"
 
-    checkpath -d -o "${GRAFANA_USER}:${GRAFANA_GROUP}" -m750 "/var/lib/grafana/dashboards"
-    checkpath -d -o "${GRAFANA_USER}:${GRAFANA_GROUP}" -m750 "/var/lib/grafana/plugins"
-    checkpath -d -o "${GRAFANA_USER}:${GRAFANA_GROUP}" -m750 "/var/lib/grafana/sessions"
+    checkpath -d -o "${GRAFANA_USER}:${GRAFANA_GROUP}" -m750 "${GRAFANA_DATADIR}"
```

Comment:
In my opinion, grafana.initd should provide the original defaults for GRAFANA_DATADIR and GRAFANA_LOGDIR in case they are unset in grafana.confd. This would also allow them to be commented by default since changing them usually won't be necessary.

Reply:
@doctaweeks Thank you very much for your suggestion! I've added default values and commented the values in confd in a second commit.